cluster/manifests/z-kubenurse/daemonset.yaml

```diff
@@ -0,0 +1,83 @@
+{{if eq .Cluster.ConfigItems.network_monitoring_enabled "true" }}
+---
+# Source: kubenurse/templates/daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    application: kubernetes
+    component: kubenurse
+  name: kubenurse
+  namespace: kubenurse
+spec:
+  selector:
+    matchLabels:
+      application: kubernetes
+      component: kubenurse
+  template:
+    metadata:
+      labels:
+        application: kubernetes
+        component: kubenurse
+      annotations:
+        prometheus.io/path: "/metrics"
+        prometheus.io/port: "8080"
+        prometheus.io/scheme: "http"
+        prometheus.io/scrape: "true"
+    spec:
+      serviceAccountName: "kubenurse"
+      containers:
+        - name: "kubenurse"
+          env:
+            - name: KUBENURSE_INGRESS_URL
+              value: https://kubenurse.{{ .Values.hosted_zone }}
+            - name: KUBENURSE_SERVICE_URL
+              value: "http://kubenurse.kubenurse.svc.cluster.local:8080"
+            - name: KUBENURSE_INSECURE
+              value: "true"
```

Comment:
If that's for calling `https://kubenurse.{{ .Values.hosted_zone }}` we might be able to set it to `false` since it serves a valid certificate. If it's for the internal API server we might be able to use `KUBENURSE_EXTRA_CA` to point it to the mounted certificate.

Reply:
the kubenurse checker has a single http client for all checks (including pod to pod communication or calling api server or calling the me_ingress)

I was more thinking of calling the pod ip may fail